k8s/overlays/azure/common/postgresql-pv-claim-azure-disk.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: azure-disk-pvc
```

Comment:
Maybe rename it to `gridcapa-postgresql-data-storage` in order to be consistent with the naming elsewhere in the project